src/VirtoCommerce.DescriptionExportImportModule.Data/Services/ProductDescriptionSearchService.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using VirtoCommerce.CatalogModule.Data.Model;
+using VirtoCommerce.CatalogModule.Data.Repositories;
+using VirtoCommerce.DescriptionExportImportModule.Core.Models;
+using VirtoCommerce.DescriptionExportImportModule.Core.Services;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Data.Infrastructure;
+
+namespace VirtoCommerce.DescriptionExportImportModule.Data.Services
+{
+    public class ProductDescriptionSearchService : IProductDescriptionSearchService
```

Comment:
Why naming does not match the domain model's name? I mean EditorialReview or just Review.